packages/pds/src/api/com/atproto/repo/createRecord.ts

```diff
@@ -52,26 +52,24 @@ export default function (server: Server, ctx: AppContext) {
         throw err
       }
 
-      await ctx.db.transaction(async (dbTxn) => {
-        const repoTxn = ctx.services.repo(dbTxn)
-        const backlinkDeletions = validate
-          ? await getBacklinkDeletions(dbTxn, ctx, write)
-          : []
-        try {
-          await repoTxn.processWrites(
-            did,
-            [...backlinkDeletions, write],
-            now,
-            swapCommitCid,
-          )
-        } catch (err) {
-          if (err instanceof BadCommitSwapError) {
-            throw new InvalidRequestError(err.message, 'InvalidSwap')
-          } else {
-            throw err
-          }
+      const backlinkDeletions = validate
+        ? await getBacklinkDeletions(ctx.db, ctx, write)
+        : []
```

Comment:
I think the backlink result here could technically get out of sync, but it may be acceptable since it's not critical functionality.  Come to think of it, I suppose that was already probably an issue to some extent 🤔

Reply:
yeah i thought about that... but my reaction was basically what you said:
- was already an issue
- not critical path
- didn't wanna have to mess with it 😅